composer.json

```diff
@@ -24,8 +24,11 @@
         "doctrine/annotations": "~1.4",
         "symfony/console": "~3.0|~4.0"
     },
+    "conflict": {
+        "symfony/yaml": "<3.4"
```

Comment:
People can really hurt themselves if they install ORM 2.6 + Yaml <3.4, so this is now here.